src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -369,8 +378,13 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
                 <search.Component model={search} />
               </>
             )}
-            {!loading && fields.length > 1 && (
-              <FieldSelector label="Field" options={fields} value={value} onChange={model.onFieldSelectorChange} />
+            {!loading && options.length > 1 && (
+              <FieldSelector
+                label="Field"
+                options={options}
+                value={String(value)}
```

Comment:
```suggestion
                value={String(value ?? '')}
```

Maybe this, to prevent `"undefined"`.

Reply:
`VariableValue` cannot be undefined, at least in the types definition:

```
type VariableValue = VariableValueSingle | VariableValueSingle[];
type VariableValueSingle = string | boolean | number | CustomVariableValue;
```

I updated the types in the `CustomConstantVariable.ts` to only allow `isMulti` as false, and hinted that the value type is `VariableValueSingle`, so we have to address this when folks want to enable multi without pushing runtime errors.